janusgraph-dist/src/assembly/static/bin/janusgraph.sh

```diff
@@ -83,7 +83,7 @@ wait_for_cassandra() {
     while [ $now_s -le $stop_s ]; do
         echo -n .
         # The \r\n deletion bit is necessary for Cygwin compatibility
-        statusbinary="`$BIN/../cassandra/bin/nodetool statusbinary 2>/dev/null | tr -d '\n\r'`"
+        statusbinary="`$BIN/../cassandra/bin/nodetool -Dcom.sun.jndi.rmiURLParsing=legacy statusbinary 2>/dev/null | tr -d '\n\r'`"
```

Comment:
Copying [my comment from the other PR](https://github.com/JanusGraph/janusgraph/pull/3033#discussion_r868954744):

This disables new validations of URL parsing which seems to have been added for security reasons. Is it really necessary to disable this?

Reply:
It was on of the tests.

Reply:
I have no idea it happened in the first place.

Reply:
@FlorianHockmann @farodin91 I think Java 8 was upgraded in CI because we didn't specify correct fixed version (which we probably should do) and it caused the issue described [here](https://community.datastax.com/questions/13764/java-version-for-cassandra-3113.html) (see the answer [here](https://community.datastax.com/answers/13817/view.html)).
So, I believe this is the correct way to fix that issue as for now but later we will need to upgrade Cassandra and remove this legacy URL parsing (looks like [related issue](https://github.com/apache/cassandra/pull/1586) was closed recently).

Reply:
Thanks for looking into this and the detailed explanation, @porunov! Then